tests/functional/codegen/modules/test_exports.py

```diff
@@ -131,9 +128,7 @@ def foo() -> uint256:
     assert c.foo() == 5
 
 
-# not sure if this one should work
```

Comment:
For the the future we should track these cases (i.e. `ambiguous spec`) somewhere properly, since we obviously forgot about it and thanks to feedback rediscovered it.

Nit: if we use `xfail`, we should also add the `reason` attribute instead of a comment to be consistent: 
- https://github.com/vyperlang/vyper/blob/e48ff3290f5490dd491cff4bde9f9a4047b88ed5/tests/functional/builtins/codegen/test_empty.py#L415
- https://github.com/vyperlang/vyper/blob/e48ff3290f5490dd491cff4bde9f9a4047b88ed5/tests/functional/builtins/codegen/test_empty.py#L459
- https://github.com/vyperlang/vyper/blob/e48ff3290f5490dd491cff4bde9f9a4047b88ed5/tests/unit/cli/vyper_json/test_get_inputs.py#L134

Reply:
i don't *really* get this -- it's easy to grep the codebase or to run with `pytest --runxfail`. i guess you mean we should be doing this on a regular basis?

Reply:
> it's easy to grep the codebase or to run with `pytest --runxfail`. i guess you mean we should be doing this on a regular basis?

My point is not that it's easy, but that we forget about certain `xfail` tests that are part of a bigger features (like in this case) and we forget to address it before the release.